### DISCUSSION TITLE
Support for .NET Core/.NET 5+ because of IKVM getting that support?

### DISCUSSION THREAD

**Discussion by vincelombardo** [sergey-tihon/Stanford.NLP.NET#119] [Q&A] [answered]
Hi, I read from your ColreNLP page that Stanford.NLP.NET is built on top of IKVM.NET which only supports .NET Framework and not .NET Core/.NET 5+

I had been watching the IKVM Revived space and recently found out that they are in prerelease for having a version with .NET Core and .NET 5 and higher (see announcement at ([IKVM is back! (now with .NET Core)](https://www.reddit.com/r/csharp/comments/v0mva8/ikvm_is_back_now_with_net_core/)) and also the [GitHub repository](https://github.com/ikvm-revived/ikvm)

With this in mind, would you think you would have plans to use this version of IKVM when it is released in order to give Stanford.NLP.NET support for .NET Core/.NET 5+?

**Comment by GeorgeS2019** [comment]
https://www.reddit.com/r/dotnet/comments/vqqpqb/ikvm_820/

> Simply include `<IkvmReference Include="mylib.jar" />` into your SDK-style MSBuild project files and let it do the magic for you

![image](https://user-images.githubusercontent.com/49812372/177466010-977f9e89-af4b-424e-ac53-52540748a2b8.png)

**Comment by GeorgeS2019** [comment]
https://github.com/ikvm-revived/ikvm/releases/tag/8.2.1

**Comment by GeorgeS2019** [comment, marked as answer]
The .NET Core versin just merged. This can be closed now
